### ISSUE TITLE
Support bzlmod

### ISSUE THREAD

**Issue by kriscfoster** [salesforce/rules_docker_compose_test#14]
Some people who would like to use this are probably already on bzlmod. We should support that before v1.0.0.

**Comment by chickenandpork** [comment]
@kriscfoster  resolved by #26 ?

**Comment by kriscfoster** [comment]
Yes good catch, forgot to close this. rules_docker_compose is in BCR now - https://registry.bazel.build/modules/rules_docker_compose_test
